ml-agents/mlagents/trainers/trainer.py

```diff
@@ -228,6 +229,7 @@ def write_summary(
                         is_training,
                     )
                 )
+                set_gauge(f"{self.brain_name}.mean_reward", mean_reward)
```

Comment:
@xiaomaogy This is what you mentioned you wanted to track. In the output json, it adds an entry like
```
  "gauges": [
    {
      "name": "HallwayLearning.mean_reward",
      "value": -0.5,
      "min": -1.0
      "max": -0.5,
      "count": 3
    }
  ],
```

Reply:
This looks exactly what I need. Is this PR ready to be merged?

Reply:
Sure, like I mentioned, I didn't add support for gauges from subprocesses yet, but that should be OK for your purpose. Let me just add a TODO.